### DISCUSSION TITLE
LDM Notes for January 18th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6894] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-01-18.md

## Agenda

- Nullable post-conditions and `async`/`await`
- Semi-colon bodies for type declarations

**Comment by HaloFour** [comment]
> Allow semicolon bodies for all type declarations.

I guess that basically shuts down proposals like #928.

**Reply by 333fred** [reply]
Those were already shut down. `record R;` is legal today.

**Reply by 333fred** [reply]
Though if we wanted to use a different syntax, we could.
